Assets/MixedRealityToolkit/StandardAssets/Shaders/InstancedColored.shader

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+Shader "Hidden/Instanced-Colored"
+{
+    Properties
+    {
+        _Color("Color", Color) = (1.0, 1.0, 1.0, 1.0)
+        _ZWrite("ZWrite", Int) = 1.0 // On
+        [Enum(UnityEngine.Rendering.CompareFunction)] _ZTest("ZTest", Int) = 4.0 // LEqual
+        [Enum(UnityEngine.Rendering.CullMode)] _Cull("Cull", Int) = 0.0 // Off
+    }
+
+    SubShader
+    {
+        Pass
+        {
+            Name "Main"
+            Tags{ "RenderType" = "Opaque" "PerformanceChecks" = "False" }
```

Comment:
Can we add some rationale on why we're disabling a performance check of this shader instead of fixing any potential warning emitted by the check?

Reply:
Good idea, I just removed that tag since it doesn't really make sense for this shader.